index.js

```diff
@@ -89,7 +94,93 @@ var PersonaClient = function (config) {
     // need to instantiate this based on the configured scheme
     this.http = require(this.config.persona_scheme);
 
-    this.debug("Persona Client Created");
+    if (this.config.cert_background_refresh !== false) {
+        this.getPublicKey(function retrievedCert() {
+            this.refreshTimerId = setInterval(function refreshCert() {
+                this.getPublicKey(function retrievedPublicKey() {
+                    log('debug', 'retrieved public key');
+                }, true);
+            }.bind(this), PUBLIC_KEY_AUTO_REFRESH_TIMEOUT);
+        }.bind(this));
+    }
+
+    this.log('debug', "Persona Client Created");
+};
+
+/**
+ * Retrieve Persona's public key that is used to sign the JWTs.
+ * @param {callback} cb function(err, publicCert)
+ * @param {boolean=} refresh (optional) refresh the certificate
```

Comment:
The term public key and certificate are being used interchangeably. It would be nice if one term was picked.